themes/docdock/layouts/partials/menu.html

```diff
@@ -1,86 +1,69 @@
-        
-        {{- if ne $numberOfPages 0 }}
-          {{- if or (.IsAncestor $currentNode) (.Params.alwaysopen) }}
-            <i class="fa fa-angle-down fa-lg category-icon"></i>
-          {{- else -}}
-            <i class="fa fa-angle-right fa-lg category-icon"></i>
-          {{- end}}
-        {{- end}}
+        "
+>
+  <div>
+    <a href="{{ .RelPermalink}}"
+      >{{safeHTML .Params.Pre}}{{.Title}}{{safeHTML .Params.Post}}</a
+    >
 
-        {{- if $showvisitedlinks}}<i class="fa fa-circle-thin read-icon"></i>{{end}}
-      </div>
-      {{- if ne $numberOfPages 0 }}
-        <ul>
-          {{- .Scratch.Set "pages" .Pages }}
-          {{- if .Sections}}
-          {{- .Scratch.Set "pages" (.Pages | union .Sections) }}
-          {{- end}}
-          {{- $pages := (.Scratch.Get "pages") }}
-
-        {{- if eq .Site.Params.orderPagesBy "title"}}
-          {{- range $pages.ByTitle }}
-            {{- if and .Params.hidden (not $.showhidden) }}
-            {{- else}}
-            {{- template "section-tree-nav" dict "sect" . "currentnode" $currentNode "showvisitedlinks" $showvisitedlinks }}
-            {{- end}}
-          {{- end}}
-        {{- else}}
-          {{- range $pages.ByWeight }}
-            {{- if and .Params.hidden (not $.showhidden) }}
-            {{- else}}
-            {{- template "section-tree-nav" dict "sect" . "currentnode" $currentNode "showvisitedlinks" $showvisitedlinks }}
-            {{- end}}
-          {{- end}}
-        {{- end}}
-        </ul>
-      {{- end}}
-    </li>
-  {{- else}}
-    {{- if not .Params.Hidden }}
-      <li data-nav-id="{{.Permalink}}" class="dd-item
-     {{- if eq .Permalink $currentNode.Permalink}} active{{end -}}
-      ">
-        <div>
-          <a href="{{ .RelPermalink}}">
-            {{safeHTML .Params.Pre}}{{.LinkTitle}}{{safeHTML .Params.Post}}
-          </a>
-          {{- if $showvisitedlinks}}<i class="fa fa-circle-thin read-icon"></i>{{end}}
-        </div>
-    </li>
-     {{- end}}
+    {{- if ne $numberOfPages 0 }} {{- if or (.IsAncestor $currentNode)
+    (.Params.alwaysopen) }}
+    <i class="fa fa-angle-down fa-lg category-icon"></i>
+    {{- else -}}
+    <i class="fa fa-angle-up fa-lg category-icon"></i>
+    {{- end}} {{- end}} {{- if $showvisitedlinks}}<i
+      class="fa fa-circle-thin read-icon"
+    ></i
+    >{{end}}
+  </div>
+  {{- if ne $numberOfPages 0 }}
+  <ul>
+    {{- .Scratch.Set "pages" .Pages }} {{- if .Sections}} {{- .Scratch.Set
+    "pages" (.Pages | union .Sections) }} {{- end}} {{- $pages := (.Scratch.Get
+    "pages") }} {{- if eq .Site.Params.orderPagesBy "title"}} {{- range
+    $pages.ByTitle }} {{- if and .Params.hidden (not $.showhidden) }} {{- else}}
+    {{- template "section-tree-nav" dict "sect" . "currentnode" $currentNode
+    "showvisitedlinks" $showvisitedlinks }} {{- end}} {{- end}} {{- else}} {{-
+    range $pages.ByWeight }} {{- if and .Params.hidden (not $.showhidden) }} {{-
```

Comment:
Seems like visual studio code messed up the formatting here, let's see if we can fix this